File: src/Views/StartView.cs
using CineComplex.Classes.Base;
using CineComplex.Interfaces;
using CineComplex.Views.FormViews;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CineComplex.Views
{
    public class StartView : AViewBase<StartView>, IView
    {
        private int _choice = 0;
        public int Choice { get => _choice; set => _choice = value; }
        public List<string> MenuList
        {
            get;
            set;
        }
        public void LoadMenuList()
        {
            StartView.Instance.MenuList = new List<string>() {
                "1. Login ",
                "2. Register ",
                "3. Forgot Password ",
                "4. Create View And ViewModel Files",
                "5. Exit",
            };
        }

        public void View()
        {
            StartView.Instance.LoadMenuList();

            do//main loop
            {
                Console.Clear();
                Console.WriteLine("\t----- !!! Salam Hindusthan !!! -----");
                Console.WriteLine("================================================");

                Console.WriteLine("\nStart - CineComplex");
                Console.WriteLine("-------------------------------------------------");

                Console.WriteLine();

                Console.WriteLine("\nMenu : ");
                Console.WriteLine("---------------");

                foreach (string instr in StartView.Instance.MenuList)
                {
                    Console.WriteLine(instr);
                }

                Console.Write("Enter Your Choice : ");


                int.TryParse(Console.ReadLine(), out _choice);
                switch (Choice)
                {
                    case 1:
                        HomeView.Instance.View();
                        break;

                    case 2:
                        ForgotPasswordFormView.Instance.View();
                        break;

                    default:
                        Console.WriteLine("Please enter the valid Choice .....");
                        break;
                }
            } while (Choice != StartView.Instance.MenuList.Count);
        }
    }
}

File: src/Views/StartView.py
from src.Classes.Base.ABaseView import ABaseView
from src.Views.FormViews.SignInFormView import SignInFormView
from src.Views.FormViews.SignUpFormView import SignUpFormView
import os
import sys


class StartView(ABaseView):
    def __init__(self):
        super().__init__()
        self.MenuList = ["1. Login", "2. Sign Up", "3. Forgot Password", "4. Exit"]
        self.Menu = {
            1: lambda: SignInFormView().View(),
            2: lambda: SignUpFormView().View(),
            # 3: lambda: ForgotPasswordFormView().View(),
        }

    def View(self):

        while self.Choice != len(self.MenuList):
            # os.system("cls")
            print("\n\t----- !!! Salam Hindusthan !!! -----")
            print("================================================")
            print("🏠 Home - CineComplex")
            print("-------------------------------------------------")
            print("\nMenu:")
            print("---------------")

            for instr in self.MenuList:
                print(instr)

            try:
                self.Choice = int(input("🎯 Enter Your Choice: ").strip())
            except Exception as e:
                print(f"Error : {e}")
                print("❌ Invalid choice. Please enter a number.")
                continue

            app = self.Menu.get(self.Choice)

            if app is not None:
                app()
